makeflow/src/makeflow.c

```diff
@@ -1523,6 +1538,14 @@ int main(int argc, char *argv[])
 				if(!wrapper) wrapper = makeflow_wrapper_create();
 				makeflow_wrapper_add_output_file(wrapper, optarg);
 				break;
+			case LONG_OPT_SHARED_FS:
+				assert(shared_fs);
+				s = string_combine(xxstrdup(optarg), "/");
+				t = realpath(s, NULL);
+				free(s);
+				if (!t) fatal("can't access shared filesystem %s: %s", optarg, strerror(errno));
+				list_push_head(shared_fs, string_combine(t, "/"));
```

Comment:
@trshaffer 
It looks like you might be appending the trailing slash twice (here and 1543)? Changing 1547 to this makes my makeflow script work:
```
list_push_head(shared_fs, t);
```